app/assets/javascripts/Components/Assessment_Chart/fraction_statistic.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export class FractionStat extends React.Component {
```

Comment:
The name of the component should be the same as the file name. I'm fine with `FractionStat`, so you should rename the file to `fraction_stat.jsx`